src/assets/scripts/client/InputController.js

```diff
@@ -711,6 +714,16 @@ export default class InputController {
     }
 
     /**
+     * Disable default behavior of right-click menu appearing on right click
+     *
+     * @for InputController
+     * @method _disableRightClickMenu
+     */
+    _disableRightClickMenu() {
+        document.querySelector('body').addEventListener('contextmenu', (e) => e.preventDefault());
```

Comment:
If we must do it this way, there are a few things that we should change:
- add a class property for `body` (see current `$window` prop) that gets set in the `_init()` method
- jquery is not necessary, we can keep it like it is here.
- move the attachment part of this to the `setupHandlers()` method
- use `event` instead of `e` it avoid confusion and add additional context to the call
- add a `removeEventListener()` call that removes this listener. this can live in the `disable()` method
- if the disable method does not exist, create one